plinth/modules/tor/templates/tor.html

```diff
@@ -26,22 +26,33 @@
   {% if config_running %}
     <meta http-equiv="refresh" content="3" />
   {% endif %}
-
+<style type="text/css">
+  .loader {
+    border: 16px solid #f3f3f3; /* Light grey */
+    border-top: 16px solid #3498db; /* Blue */
+    border-radius: 50%;
+    width: 120px;
+    height: 120px;
+    animation: spin 2s linear infinite;
+}
+
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+</style>
 {% endblock %}
 
 
 {% block configuration %}
 
-  <h3>{% trans "Status" %}</h3>
```

Comment:
Is there a reason to remove "Status" here and add it in 2 places below? It seems like there would be no difference in the final display.